stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/extensions/ViewGroup.kt

```diff
@@ -0,0 +1,7 @@
+package io.getstream.chat.android.ui.utils.extensions
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+
+internal val ViewGroup.inflater: LayoutInflater
```

Comment:
This won't be needed after https://github.com/GetStream/stream-chat-android/pull/1003